lib/srv/desktop/rdp/rdpclient/src/cliprdr.rs

```diff
@@ -103,38 +103,32 @@ impl Client {
     /// update_clipboard is invoked from Go.
     /// It updates the local clipboard cache and returns the encoded message
     /// that should be sent to the RDP server.
+    ///
+    /// The data passed to this function should be valid UTF-8 text.
     pub fn update_clipboard(&mut self, data: Vec<u8>) -> RdpResult<Vec<Vec<u8>>> {
```

Comment:
If broken utf 8 will result in an error, why not just take a `String` here instead?